mmv1/products/firestore/api.yaml

```diff
@@ -17,9 +17,115 @@ versions:
   - !ruby/object:Api::Product::Version
     name: ga
     base_url: https://firestore.googleapis.com/v1/
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://firestore.googleapis.com/v1/
 scopes:
   - https://www.googleapis.com/auth/cloud-platform
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Google Cloud Firestore API
+    url: https://console.cloud.google.com/apis/library/firestore.googleapis.com
 objects:
+  - !ruby/object:Api::Resource
```

Comment:
q: Do we need another implementation for support `import Terraform configuration from the existing GCP project`? Or this is auto generated after we implement the surface

Reply:
Well, importing does actually require a bit of handwriting in this case, but I have already implemented it. So it should already work.